### PR TITLE
Tweaks ore proccessing machinery icons

### PR DESCRIPTION
**Pull request by Shadow-Quill**
## About The Pull Request

Furnace now has an inactive sprite, stacker will only do the stacking animation when it ejects a sheet (excepting slag, it should do so either when manually ejected or automatically ejected)

Closes https://github.com/discordia-space/CEV-Eris/issues/1730

## Why It's Good For The Game

More intuitive sprite behaviour

## Changelog
:cl:
imageadd: Added an off-state for the ore smelter.
tweak: Sheet stacker machine only does it's stacking animation when it releases a stack of sheets.
/:cl:

### CODE REVIEW

**Comment by Shadow-Quill** [comment]
as for why I've got `d9cbdb6` committed, it's because otherwise I'd be including a deleted file with each PR. Whee.

**Comment by Shadow-Quill** [comment]
CI failure caused by runtimes fixed in https://github.com/discordia-space/CEV-Eris/pull/6909

**Comment by Kurgis** [comment]
An Eris downstream contributing back upstream? I'm in fear they'll just realize they're on the wrong repo and stop, otherwise this must be a Christmas Miracle.

**Comment by Shadow-Quill** [comment]
> An Eris downstream contributing back upstream? I'm in fear they'll just realize they're on the wrong repo and stop, otherwise this must be a Christmas Miracle.

My ADHD-riddled brain has focused on wanting to fix up issues here. So, essentially yes.

**Comment by Kurgis** [comment]
> My ADHD-riddled brain has focused on wanting to fix up issues here. So, essentially yes.

Thank you.

**Review by Firefox13 — APPROVED** [review]

File: code/modules/mining/machine_stacking.dm [hunk 1]
```diff
@@ -98,6 +98,7 @@
 		stack_storage[material_name] -= amount
 		var/stacktype = material_stack_type(material_name)
 		new stacktype (get_step(src, output_dir), amount)
+		FLICK("stacker_eject", src)
```
Comment (SirRichardFrancis) [inline code/modules/mining/machine_stacking.dm:101]: This proc have been removed in #6932, wonder how this got a pass. @Firefox13

**Pull request merged**